### DISCUSSION TITLE
How to load OMZ plugin that contains `_<name>` file instead of `<name>.plugin.zsh`?

### DISCUSSION THREAD

**Discussion by seblemaguer** [marlonrichert/zsh-snap#104] [Q&A] [answered]
Hello,

sorry for the stupid question but since the last commit, I am facing the following issue:

```
znap source: file not found: /home/lemagues/.cache/znap/ohmyzsh/plugins/docker
```

the minimal zshrc is the following

```sh
#!/bin/zsh

# Load znap
source "$HOME/.cache/znap/zsh-snap//znap.zsh"
znap source ohmyzsh/ohmyzsh plugins/docker
```

I have the same issue by replacing docker by ripgrep. It seems to be impacted when the subdirectory doesn't contain any <file>.plugins.sh but a file following the pattern _file.

Am I using the wrong command?

**Comment by marlonrichert** [comment, marked as answer]
This has been answered here: https://github.com/marlonrichert/zsh-snap/discussions/55#discussioncomment-502336

**Comment by seblemaguer** [comment]
thanks a lot

**Reply by marlonrichert** [reply]
FYI: I just pushed in a new feature. You can now install completion functions from the command line with `znap install`. For example, to install _all_ completion functions that ship with Oh-My-Zsh, just run the following:
```zsh
znap install ohmyzsh/ohmyzsh
```

**Reply by seblemaguer** [reply]
wow, that sounds pretty neat :)
